src/OnboardingSPA/components/LivePreview/BlockPreview/index.js

```diff
@@ -54,7 +54,7 @@ const BlockPreview = ( {
 		} else {
 			setSettings( storedPreviewSettings );
 		}
-	}, [] );
+	}, [ previewSettings, storedPreviewSettings  ] );
```

Comment:
Will it still work if we remove the `storedPreviewSettings` dependency?

Reply:
Removed.